### ISSUE TITLE
Compilation issue

### ISSUE THREAD

**Issue by maximecperoumal** [infinitellamas/Arduino-Tracker-Plugin#5]
I follow a friend's instruction to build the DLL but OSVR failed to load it. This is what I've done: https://docs.google.com/document/d/1X5gKvR7drLpHa3ATR-cu2qKQfTZvvQISbzVddeFd7WM/edit?usp=sharing

Where do you think is the problem ?

**Comment by terminal29** [comment]
Did you add the options for the plugin into your osvr_server_config.json file? Your build process looks fine but it is hard to tell what is wrong without knowing what the error actually is.

**Comment by maximecperoumal** [comment]
The compalitation process is fine but when launching OSVR with the plugin it failed to load.

**Comment by gabriel-combe** [comment]
Yes, I have the same problem, when I launch osvr_server, it give this : 
![image](https://user-images.githubusercontent.com/26323828/36631134-a5c2e28a-1972-11e8-9b97-6c41103127c4.png)

And my osvr_server_config.json contains this : 
[osvr_server_config.txt](https://github.com/terminal29/Arduino-Tracker-Plugin/files/1754406/osvr_server_config.txt)

I hope you can help us.

**Comment by terminal29** [comment]
Does the dll from the latest release work on your system?
